indexer/osm_editor.cpp

```diff
@@ -426,6 +426,25 @@ bool Editor::IsCreatedFeature(FeatureID const & fid)
   return fid.m_index >= kStartIndexForCreatedFeatures;
 }
 
+bool Editor::WasDefaultNameSaved(FeatureID const & fid) const
+{
+  if (IsCreatedFeature(fid))
+    return false;
+
+  if (FeatureStatus::Created == GetFeatureStatus(fid))
+    return false;
+
+  auto const originalFeaturePtr = m_getOriginalFeatureFn(fid);
+  if (!originalFeaturePtr)
+  {
+    LOG(LERROR, ("A feature with id", fid, "cannot be loaded."));
+    alohalytics::LogEvent("Editor_MissingFeature_Error");
+    return false;
+  }
+
+  return originalFeaturePtr->HasName();
```

Comment:
Moreover, it's possible for a feature to have a specific name, but do not have a default name.